subgraphs/saucerswap/v2/src/mappings/position-manager.ts

```diff
@@ -3,36 +3,44 @@ import {
   Collect,
   DecreaseLiquidity,
   IncreaseLiquidity,
-  NonfungiblePositionManager,
-  Transfer
+  NonfungiblePositionManager
 } from '../types/NonfungiblePositionManager/NonfungiblePositionManager'
 import { Bundle, Position, PositionSnapshot, Token } from '../types/schema'
 import { ADDRESS_ZERO, factoryContract, ZERO_BD, ZERO_BI } from '../utils/constants'
-import { Address, BigInt, ethereum } from '@graphprotocol/graph-ts'
+import { Address, BigInt, ethereum, log } from '@graphprotocol/graph-ts'
 import { convertTokenToDecimal, loadTransaction } from '../utils'
 
+const excludedBlocks: BigInt[] = [
```

Comment:
Q: do we still need to exclude and if so we should say why these blocks are being excluded.
Could the missing blocks ever be reindexed?

Reply:
I can try after the improvements we've done to see if we don't need this anymore. 👍

Reply:
I did took another look at this and was able to find the culprit of some blocks failing, the issue was fixed on this PR of the relay:
https://github.com/hashgraph/hedera-json-rpc-relay/pull/2333

It was deployed to `mainnet` and I have re-indexed the subgraph and now is working without the need to hardcode "faulty" blocks.